src/constants/howConst.js

```diff
@@ -1,3 +1,11 @@
+import phoneImg from '../images/phone.svg';
+import containerImg from '../images/container-monster.svg';
+import truckImg from '../images/truck.svg';
+import starsImg from '../images/stars.svg';
+import curveArrow from '../images/up-right-arrow.svg';
+import straightArrow from '../images/straight-arrow.svg';
+import mobile180Arrow from '../images/mobile-u-arrow.svg';
+
 export const HowData = {
   title: 'How it works?',
```

Comment:
This just occurred to me: 'how it works?' is not good grammar.  I feel that there shouldn't be a question mark there.  I know the design has one, but it doesn't make any sense

Reply:
I agree, I will change it